src/main/java/core/basesyntax/garage/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.garage;
+
+public class Excavator extends Machine{
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
```

Comment:
The '@Override' annotation should be directly above the method signature without an empty line separating them.